Assets/_BForBoss/_Core/Scripts/Labor/LaborSystem.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+
+namespace BForBoss.Labor
+{
+    public class LaborSystem
+    {
+        public Action onLaborCompleted;
+        public Action onQueueCompleted;
```

Comment:
Since this hasn't been decided yet, we don't need these Actions just yet. could just delete them for now.

Reply:
Sure, I'm using the queue one to facilitate tests but I could make an exposed IsComplete boolean flag or something to test against instead

Reply:
An idea i had, was to have an `event Action`, where it returns an enum that signifies an effect.
so something like
```
enum LaborSystemEffect {
   case LaborCompleted
   case QueueComplete
}

event Action<LaborSystemEffect> LaborSystemEffect
```

but this is for later. you can also check that `CurrentLaborSystem == null`

Reply:
That makes sense yeah. The more I think about too though I think having an IsComplete property seems useful